server/manager/src/test/java/org/apache/accumulo/manager/compaction/CompactionCoordinatorTest.java

```diff
@@ -129,6 +126,13 @@ protected void startCompactionCleaner(ScheduledThreadPoolExecutor schedExecutor)
     @Override
     protected void startRunningCleaner(ScheduledThreadPoolExecutor schedExecutor) {}
 
+    @Override
+    protected void startIdleCompactionWatcher() {
+      // This is called from CompactionCoordinator.run(). Setting shutdown to true
+      // here will exit the loop in run()
```

Comment:
I think this comment needs to be updated now that this is no longer setting a boolean

Reply:
Updated comment in f5fedcd